### ISSUE TITLE
[Development Branch] You can sometimes go through terrain

### ISSUE THREAD

**Issue by EliteAsian123** [andyhall/noa#87]
![glitch](https://user-images.githubusercontent.com/29520859/66269049-dceb3180-e811-11e9-9dca-c61bf5c12a91.gif)
From my testing, it only happens on chunk boundaries. I'm not too sure if this is a thing in the master branch. Thanks!

**Comment by EliteAsian123** [comment]
It's a really weird bug because it only happens sometimes. I'll try it on example worlds and see if it is just me!

**Comment by EliteAsian123** [comment]
I've been trying for half an hour, and I couldn't get it to happen on the example worlds *except* this one time I did it by mistake, and I couldn't do it again. Soooo yeah.

**Comment by andyhall** [comment]
Hmm, I'm not sure what it might be. Chunk borders are prone to rendering bugs, due to the overlapping voxels stored for meshing purposes, but the physics system is pretty much isolated from the chunking abstraction so it doesn't usually have issues like that.

One thing I can offer, the physics system is written such that collisions only happen as a body *enters* a solid voxel. The engine doesn't check whether a body starts a given tick already overlapping a solid voxel, so if you do something like manually positioning the player somewhere, and maybe rounding issues cause the player to start off just inside a voxel, that might cause something similar...

**Comment by IceHacks** [comment]
Happened to me when using the `master` branch, but only once (I jumped and went through a block). Thanks for your time @andyhall. Not much of an issue for me. But I was wondering how to change the fill of the `targetedBlock` outline. Or just what object path contains the outline? Thanks again c:

**Comment by EliteAsian123** [comment]
@IceHacks I believe you can disable the outline and create your own, however I'm not too sure how you could create the outline.

**Comment by andyhall** [comment]
Closing this as it's a few versions ago, please file a new issue if it reproduces.
